src/index.js

```diff
@@ -1,11 +1,17 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable */ 
```

Comment:
Kindly avoid disabling linter checks inline. They represent guidelines you share with your team and most of the time help to ensure adherence to best coding practices.

Reply:
This change has been done well